.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ jobs:
   # Run Linting
   lint-all:
     docker:
-      - image: circleci/node:12.14.1-buster-browsers
+        - image: circleci/node:12.16.1-buster-browsers
```

Comment:
Just hail Mary attempts at fixing with some tests that fail on CI and not locally on my dev machine.